server/container/guice/jpa-guice/src/main/java/org/apache/james/modules/mailbox/JPAMailboxModule.java

```diff
@@ -101,7 +102,7 @@ protected void configure() {
         bind(Authenticator.class).to(UserRepositoryAuthenticator.class);
         bind(MailboxManager.class).to(OpenJPAMailboxManager.class);
         bind(StoreMailboxManager.class).to(OpenJPAMailboxManager.class);
-        bind(SessionProviderImpl.class).in(Scopes.SINGLETON);
```

Comment:
Please keep the singletong binding...

Reply:
Still present, the problem was a bad copy/paste with twice this singleton binding, wich resulted in the interface to implementation binding missing